<?xml version='1.0' encoding='utf-8'?>
<sdf version="1.5">
    <!--Generated on demand-->
    <model name="goal">
        <static>true</static>
        <link name="sphere_inner">
            <pose>3.029200639895506 0.0 0.7227488102424202 0 0 0</pose>
            <visual name="visual">
                <geometry>
                    <sphere>
                        <radius>0.001</radius>
                    </sphere>
                </geometry>
                <transparency>1.0</transparency>
            </visual>
        </link>
        <link name="sphere_outter">
            <pose>3.029200639895506 0.0 0.7227488102424202 0 0 0</pose>
            <visual name="visual">
                <geometry>
                    <sphere>
                        <radius>0.001</radius>
                    </sphere>
                </geometry>
                <transparency>1.0</transparency>
            </visual>
        </link>
    </model>
</sdf>
